### PR TITLE
Added new command that will return the days until an event (events based off an object of events with date formats)

### PR DESCRIPTION
**Pull request by N3rdP1um23**

### CODE REVIEW

File: lib/handlers/messages.ts [hunk 1]
```diff
@@ -1,707 +1,751 @@
...
+
+	// Tableflip Reaction
+	if(m.startsWith('(╯°□°）╯︵ ┻━┻') || m.startsWith('/tableflip')) {
+		// Unflip the table
+		msg.reply('┬─┬ ノ( ゜-゜ノ)');
+	}
+
+	// Days until message response
+	// This is mainly for holidays, however, it can be adapted for multiple dates provided they're listed in the events object
+	if(m.startsWith('!daysUntil')) {
```
Comment (TimmyRB) [inline lib/handlers/messages.ts:224]: Add `return` statement to command when successfully executed to stop Bot from showing ❓ Reaction

File: lib/funcs.ts [hunk 2]
```diff
@@ -34,6 +34,8 @@ export function help(member: Discord.GuildMember, page: number, channel?: Discor
         tips.push({ 'name': 'Warn User(s)', 'value': '```!warn #rule_num @non_admin_user[]```' })
     }
 
+    tips.push({ 'name': 'Get days until an event (mainly holiday)', 'value': '```!daysUntil event name```'})
```
Comment (TimmyRB) [inline lib/funcs.ts:37]: Move this beside where the other Tips are set that also do not require special permissions to have this appear before Admin Commands

File: lib/handlers/messages.ts (hunk 1 above)
Comment (TimmyRB) [inline lib/handlers/messages.ts:224]: Add `return` statement to command when successfully executed to stop Bot from showing ❓ Reaction

**Review by TimmyRB — CHANGES_REQUESTED** [review]

File: lib/funcs.ts (hunk 2 above)
Comment (N3rdP1um23) [reply lib/funcs.ts]: Done!

File: lib/handlers/messages.ts (hunk 1 above)
Comment (N3rdP1um23) [reply lib/handlers/messages.ts]: Done!

**Review by TimmyRB — APPROVED** [review]

**Pull request merged**